### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by satellitestudiodesign**

### CODE REVIEW

File: src/features/map/MapControls.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ const MapControls = ({ bounds }: { bounds: MiniglobeBounds | null }) => {
   const [showCoords, setShowCoords] = useState(false)
   const [pinned, setPinned] = useState(false)
   const [showDMS, setShowDMS] = useState(false)
+  console.log(latitude, longitude, bounds)
```
Comment (nerik) [inline src/features/map/MapControls.tsx:17]:
```suggestion

```

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400&display=swap');
```
Comment (nerik) [inline src/index.css:1]: Can we avoid using the Google Fonts API and its unnecessary tracking?
I suggest uploading the font to a static GCloud bucket.

File: src/features/map/MapControls.tsx (hunk 1 above)
Comment (satellitestudiodesign) [reply src/features/map/MapControls.tsx]: Done

File: src/index.css (hunk 2 above)
Comment (satellitestudiodesign) [reply src/index.css:1]: It won't matter if we do it just for this project, we have the same call in the wordpress, the map, carrier-portal... and we have the tag manager in all as well :/

**Pull request merged**